Reduce strict error for QueryWrapper

There's currently a strict Typescript error in `QueryWrapper.tsx`.

```log
public/app/features/alerting/unified/components/rule-editor/QueryWrapper.tsx(82,9): error TS2769: No overload matches this call.
  Overload 1 of 2, '(props: Readonly<Props<DataQuery>>): QueryEditorRow<DataQuery>', gave the following error.
    Type '(query: AlertQuery) => void' is not assignable to type '(query: DataQuery) => void'.
      Types of parameters 'query' and 'query' are incompatible.
        Type 'DataQuery' is missing the following properties from type 'AlertQuery': datasourceUid, model
  Overload 2 of 2, '(props: Props<DataQuery>, context?: any): QueryEditorRow<DataQuery>', gave the following error.
    Type '(query: AlertQuery) => void' is not assignable to type '(query: DataQuery) => void'.
```

Related to #32142

--- a/public/app/features/alerting/unified/components/rule-editor/QueryWrapper.tsx
+++ b/public/app/features/alerting/unified/components/rule-editor/QueryWrapper.tsx
@@ -79,7 +79,7 @@ export const QueryWrapper: FC<Props> = ({
         query={cloneDeep(query.model)}
         onChange={(query) => onChangeQuery(query, index)}
         onRemoveQuery={onRemoveQuery}
-        onAddQuery={onDuplicateQuery}
+        onAddQuery={() => onDuplicateQuery(cloneDeep(query))}
         onRunQuery={onRunQueries}
         queries={queries}
         renderHeaderExtras={() => renderTimePicker(query, index)}

